### ISSUE TITLE
Interface renders white font on white backgroud

### ISSUE THREAD

**Issue by MaelStor** [fredldotme/harbour-owncloud#35]
![ghoustcloud_2](https://user-images.githubusercontent.com/11753780/64049358-d6afc880-cb74-11e9-8889-2516538fa3b3.png)

This renders the application completely useless since I can't see anything.

I've installed ghostcloud via snap and tried already different color schemes in System Settings -> Color and disabled the option `Apply colors to non-Qt applications` with no effect.

```
$ snap version
snap    2.40
snapd   2.40
series  16
ubuntu  18.04
kernel  5.0.0-25-generic
```

```
$ snap info ghostcloud
...
snap-id:      7Z4UghGgKX4VOE4khjdtjtqSk040ytcq
tracking:     stable
refresh-date: today at 21:07 CEST
channels:
  stable:    0.9.5 2019-08-06 (345) 158MB -
  candidate: 0.9.1 2019-05-04 (195) 159MB -
  beta:      ↑                            
  edge:      0.9.5 2019-08-10 (355) 158MB -
installed:   0.9.5            (345) 158MB -
```

```
$ plasmashell -v
plasmashell 5.12.8
```

```
$ kded5 -v
kded5 5.44.0
```

**Comment by fredldotme** [comment]
May I ask you which distribution you are using? Is it KDE Neon?
On my KDE Neon installation it renders just fine. Which hardware are you using?

Also, if you are not afraid to build from source you might want to so (with the correct CONFIG additions, findable in the README).

**Comment by MaelStor** [comment]
Thanks for reply. I am using plain Kubuntu 18.04 running in a VirtualBox with guest additions and extension pack installed. Desktop is Plasma and KDE. I'll give building myself a try and report back.

```
$ uname -a
Linux kronos 5.0.0-25-generic #26~18.04.1-Ubuntu SMP Thu Aug 1 13:51:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by fredldotme** [comment]
To contribute and try the source the recommended way is through the upstream Qt releases (download from qt.io). Then checkout the source recursively from git. Ensure you have a supported 5.9.x based Qt installation available.

TL;DR:
```
export PATH=/path/to/Qt/5.9.x/gcc_64/bin:$PATH
git clone --recursive https://github.com/fredldotme/harbour-owncloud
mkdir build-gc
cd build-gc
qmake ../harbour-owncloud/harbour-owncloud.pro CONFIG+=noadditionals CONFIG+=quickcontrols
make -j`nproc`
```

**Comment by MaelStor** [comment]
Thanks :) I didn't ever built a qt application. This helps a lot.
